lib/common/cpu.h

```diff
@@ -36,7 +36,7 @@ MEM_STATIC ZSTD_cpuid_t ZSTD_cpuid(void) {
     U32 f1d = 0;
     U32 f7b = 0;
     U32 f7c = 0;
-#ifdef _MSC_VER
+#if defined(_MSC_VER) && defined(_M_X64)
```

Comment:
The comment says that `__cpuid()` function is available for both x86 and x64,
but the test only check for x64.
Is that intentional ?